changelog.txt

```diff
@@ -40,6 +40,8 @@
 * Fix - ECE checkout error when using extensions that reduce total cart amount (eg: Gift Cards)
 * Add - Add WooCommerce Pre-Orders support to ACSS.
 * Update - Remove unused express checkout button tracking.
+* Add - Disable unsupported payment methods in Stripe settings.
```

Comment:
Is this other entry intended?

Reply:
I had just added a `.` (full stop) at the end of this entry. But no idea why it shows as a new entry here. Maybe messed up in the merge process 🤔